sdk/src/main/java/com/bugsnag/android/Client.java

```diff
@@ -511,124 +506,75 @@ public void setAutoCaptureSessions(boolean autoCapture) {
     }
 
     /**
-     * Set details of the user currently using your application.
-     * You can search for this information in your Bugsnag dashboard.
-     * <p/>
-     * For example:
-     * <p/>
-     * client.setUser("12345", "james@example.com", "James Smith");
-     *
-     * @param id    a unique identifier of the current user (defaults to a unique id)
-     * @param email the email address of the current user
-     * @param name  the name of the current user
-     */
-    public void setUser(String id, String email, String name) {
-        setUserId(id);
-        setUserEmail(email);
-        setUserName(name);
-    }
-
-    /**
-     * Removes the current user data and sets it back to defaults
+     * @return the current user information associated with the Bugsnag {@link Client}
      */
-    public void clearUser() {
-        user.setId(deviceData.getUserId());
-        user.setEmail(null);
-        user.setName(null);
-
-        SharedPreferences sharedPref =
-            appContext.getSharedPreferences(SHARED_PREF_KEY, Context.MODE_PRIVATE);
-        sharedPref.edit()
-            .remove(USER_ID_KEY)
-            .remove(USER_EMAIL_KEY)
-            .remove(USER_NAME_KEY)
-            .apply();
-        notifyBugsnagObservers(NotifyType.USER);
+    public User getUser() {
```

Comment:
Would be nice to add `@see #setUser(User)` to the JavaDoc

Reply:
👍